### ISSUE TITLE
Spyder crashes after minimizing to Start bar and reactivating the window (Windows 7)

### ISSUE THREAD

**Issue by leodevisser** [spyder-ide/spyder#8009]
### What steps will reproduce the problem?

Python 3.7.0, Spyder 3.3.1., Windows 7 (6.1.7601 Sp1).

Whenever I minimize the program to the Start bar, then try to reactivate the window, Spyder crashes. I donot have Just In Time debugging on the machine I use, but the problem seems to come from an unknown exception within Spyder.
This has consistently happened to me on Anaconda 3-5.2.0, and after removing Anaconda and reinstalling Spyder from pip3.

## Versions

* Spyder version: 3.3.1 
* Python version: 3.7.0
* Qt version: 5.9.3
* PyQt5 version: 5.9.2
* Operating System: Windows 7

### Dependencies

```
pyflakes >=0.6.0  :  2.0.0 (OK)
pycodestyle >=2.3 :  2.4.0 (OK)
pygments >=2.0    :  2.2.0 (OK)
sphinx >=0.6.6    :  1.8.1 (OK)
rope >=0.9.4      :  0.11.0 (OK)
jedi >=0.9.0      :  0.13.0 (OK)
nbconvert >=4.0   :  5.4.0 (OK)
pandas >=0.13.1   :  None (NOK)
numpy >=1.7       :  1.15.2 (OK)
sympy >=0.7.3     :  None (NOK)
cython >=0.21     :  None (NOK)
qtconsole >=4.2.0 :  4.4.1 (OK)
IPython >=4.0     :  7.0.1 (OK)
matplotlib >=2.0.0:  3.0.0 (OK)
pylint >=0.25     :  2.1.1 (OK)
```

**Comment by ccordoba12** [comment]
This error will be fixed in our next release (3.3.2). In the meantime, please use Python 3.6 and not 3.7.

**Comment by leodevisser** [comment]
Problem persists using pip and python 3.6.6. Which 3.6 should I install so I don't get this error anymore?
